airflow/contrib/operators/slack_webhook_operator.py

```diff
@@ -55,6 +57,7 @@ def __init__(self,
                  http_conn_id=None,
                  webhook_token=None,
                  message="",
+                 attachments="",
```

Comment:
Shouldn't have `attachments` a default value of type list? Probably `None`

Reply:
i think a default of None here makes the most sense. I will update